x-pack/plugins/aiops/public/components/document_count_content/document_count_chart/document_count_chart.tsx

```diff
@@ -117,46 +133,120 @@ export const DocumentCountChart: FC<Props> = ({
       from: startRange,
       to: startRange + interval,
     };
-    timefilterUpdateHandler(range);
+
+    if (viewMode === VIEW_MODE.ZOOM) {
+      timefilterUpdateHandler(range);
+    } else {
+      if (
+        typeof startRange === 'number' &&
+        originalWindowParameters === undefined &&
+        windowParameters === undefined &&
+        adjustedChartPoints !== undefined
+      ) {
+        const wp = getWindowParameters(
+          startRange + interval / 2,
+          xDomain.min,
+          xDomain.max + interval
+        );
+        setOriginalWindowParameters(wp);
+        setWindowParameters(wp);
+        brushSelectionUpdateHandler(wp);
+      }
+    }
   };
 
   const timeZone = getTimezone(uiSettings);
 
+  const [originalWindowParameters, setOriginalWindowParameters] = useState<
+    WindowParameters | undefined
+  >();
+  const [windowParameters, setWindowParameters] = useState<WindowParameters | undefined>();
+
+  function onWindowParametersChange(wp: WindowParameters) {
+    setWindowParameters(wp);
+    brushSelectionUpdateHandler(wp);
+  }
+
+  const [mlBrushWidth, setMlBrushWidth] = useState<number>();
+  const [mlBrushMarginLeft, setMlBrushMarginLeft] = useState<number>();
+
+  useEffect(() => {
+    if (viewMode !== VIEW_MODE.BRUSH) {
+      setOriginalWindowParameters(undefined);
+      setWindowParameters(undefined);
+    }
+  }, [viewMode]);
+
+  const isBrushVisible =
+    originalWindowParameters && mlBrushMarginLeft && mlBrushWidth && mlBrushWidth > 0;
+
   return (
-    <div style={{ width: width ?? '100%' }} data-test-subj="aiopsDocumentCountChart">
-      <Chart
-        size={{
-          width: '100%',
-          height: 120,
-        }}
-      >
-        <Settings
-          xDomain={xDomain}
-          onBrushEnd={onBrushEnd as BrushEndListener}
-          onElementClick={onElementClick}
-          theme={chartTheme}
-          baseTheme={chartBaseTheme}
-        />
-        <Axis
-          id="bottom"
-          position={Position.Bottom}
-          showOverlappingTicks={true}
-          tickFormat={(value) => xAxisFormatter.convert(value)}
-          timeAxisLayerCount={useLegacyTimeAxis ? 0 : 2}
-          style={useLegacyTimeAxis ? {} : MULTILAYER_TIME_AXIS_STYLE}
-        />
-        <Axis id="left" position={Position.Left} />
-        <BarSeries
-          id={SPEC_ID}
-          name={seriesName}
-          xScaleType={ScaleType.Time}
-          yScaleType={ScaleType.Linear}
-          xAccessor="time"
-          yAccessors={['value']}
-          data={adjustedChartPoints}
-          timeZone={timeZone}
-        />
-      </Chart>
-    </div>
+    <>
+      {isBrushVisible && (
+        <div className="aiopsHistogramBrushes">
+          <DualBrush
```

Comment:
Not needed for this PR, but could be useful to display the time range of the brush in a tooltip?